redaxo/src/addons/structure/lib/navigation.php

```diff
@@ -48,12 +52,12 @@ class rex_navigation
 
     private function __construct()
     {
-        // nichts zu tun
+        $this->activeLiClass = $this->activeClass;
+        $this->activeAClass = $this->activeClass;
+        $this->currentLiClass = $this->activeClass;
+        $this->currentAClass = $this->activeClass;
```

Comment:
Ich wollte im construct erstmal nur den "initialen Stand" beibehalten.

setActiveClass() und setCurrentClass() setzen den default Wert um. Da ist natürlich jetzt in Zeile 57 und 58 ein Fehler. Da muss es currentClass heissen.

Dann dachte ich mir jedoch, das es zwischen LI und A unterschiede geben sollte. Das Naming kann natürlich geändert werden, dass ganze ist eher ein "Schnellschuss", musste  flott was anpassen bei mir.